integration_tests/mockApis/responses/temporaryAbsences.ts

```diff
@@ -0,0 +1,30 @@
+export default [
+  {
+    firstName: 'John',
+    lastName: 'Doe',
+    dateOfBirth: '1971-01-01',
+    prisonNumber: 'G0013AB',
+    reasonForAbsence: 'hospital',
+  },
+  {
+    firstName: 'Karl',
+    lastName: 'Offender',
+    dateOfBirth: '1985-01-01',
+    prisonNumber: 'G0015GD',
+    reasonForAbsence: 'court appearance',
```

Comment:
we found out earlier that court appearances aren't considered internal moves

Reply:
Ah ok - I'll change this